terraform/aws/analytical-platform-development/cluster/efs-access-points.tf

```diff
@@ -3,6 +3,8 @@
 ##################################################
 
 resource "aws_efs_access_point" "eks_user_homes" {
+  #checkov:skip=CKV_AWS_329: requires a destroy to implement
+  #checkov:skip=CKV_AWS_330: requires ClientRootAccess IAM permission to implement
```

Comment:
Where does this come from? I'm not saying we should address this, just curious if this was an error back from Terraform

Reply:
No error from Terraform - just alerts from checkov 

when i tried the fix for CKV_AWS_329 the plan came back with a destroy which we agreed that we would not implement any destructive fixes and skip

As for CKV_AWS_330 requires "Enforcing a user identity is subject to the ClientRootAccess IAM permission." which we agreed we do not want to do